arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+          "$ref": "#/definitions/ProtectionContainer",
+          "description": "Resource properties"
+        }
+      }
+    },
+    "ProtectionContainer": {
+      "description": "Base class for container with backup items. Containers with specific workloads are derived from this class.",
+      "type": "object",
+      "properties": {
+        "friendlyName": {
+          "description": "Friendly name of the container.",
+          "type": "string"
+        },
+        "backupManagementType": {
+          "description": "Type of backup managemenent for the container. Possible values: AzureIaasVM, DPM or MAB",
+          "type": "string"
+        },
+        "registrationStatus": {
+          "description": "Status of registration of the container with the Recovery Services Vault.",
+          "type": "string"
+        },
+        "healthStatus": {
+          "description": "Status of health of the container.",
+          "type": "string"
+        },
+        "containerType": {
+          "description": "Type of the container. Possible values are: IaasVMContainer, Windows.",
+          "type": "string"
+        },
+        "protectableObjectType": {
+          "description": "Type of the protectable object associated with this container. Possible values: IaasVMContainer, MABWindowsContainer, AzureSqlContainer.",
+          "type": "string"
+        }
+      },
+      "discriminator": "containerType"
+    },
+    "MabContainer": {
+      "description": "Container with items backed up using MAB backup engine.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProtectionContainer"
+        }
+      ],
+      "properties": {
+        "canReRegister": {
+          "description": "Can the container be registered one more time.",
+          "type": "boolean"
+        },
+        "containerId": {
+          "format": "int64",
+          "description": "ContainerID represents the container.",
+          "type": "integer"
+        },
+        "protectedItemCount": {
+          "format": "int64",
+          "description": "Number of items backed up in this container.",
+          "type": "integer"
+        },
+        "agentVersion": {
+          "description": "Agent version of this container.",
+          "type": "string"
+        },
+        "extendedInfo": {
+          "$ref": "#/definitions/MabContainerExtendedInfo",
+          "description": "Additional information for this container"
+        }
+      },
+      "x-ms-discriminator-value": "MABWindowsContainer"
+    },
+    "MabContainerExtendedInfo": {
+      "description": "Additional information of the container",
+      "type": "object",
+      "properties": {
+        "lastRefreshedAt": {
+          "format": "date-time",
+          "description": "Time stamp when this container was refreshed.",
+          "type": "string"
+        },
+        "backupItemType": {
+          "description": "Backup items associated with this container.",
```

Comment:
description seems incorrect. Property name suggests that it is some type. If so then should this be an enum?